build.gradle

```diff
@@ -44,8 +44,8 @@ dependencies {
     compile "io.dropwizard:dropwizard-auth:${dropwizardVersion}"
     compile "io.dropwizard:dropwizard-client:${dropwizardVersion}"
     compile "de.thomaskrille:dropwizard-template-config:1.4.0"
-    // compile "io.dropwizard:dropwizard-jdbi:${dropwizardVersion}"
-    // compile ':ojdbc6_g:'
+     compile "io.dropwizard:dropwizard-jdbi:${dropwizardVersion}"
```

Comment:
This line and the one below it don't line up with the "compile" lines above them. Looks like an extra space.